.ci/magic-modules/generate-inspec.sh

```diff
@@ -25,6 +24,8 @@ if [ -z "$INSPEC_COMMIT_MSG" ]; then
 fi
 
 pushd "build/inspec"
+# We are not ready to overwrite changelog yet.
+git checkout CHANGELOG.md
```

Comment:
best to specify where to check it out from, like `git checkout HEAD -- CHANGELOG.md`.  This seems silly but you'd be surprised what can happen with git - in this case, if someone ever pushes a branch called CHANGELOG.md, you'd wind up in an absurd (and confused) state.